spec/features/staff/fund_managers_can_view_an_activity_as_xml_spec.rb

```diff
@@ -19,10 +19,14 @@
         expect(xml.at("iati-activity/iati-identifier").text).to eq(activity.identifier)
 
         # The funding organisation XML is present
-        expect(xml.at("iati-activity/participating-org/@ref").text).to eq(activity.funding_organisation_reference)
```

Comment:
Is it worth mentioning in the commit message why this changed?

Reply:
Good call, I will update the commit message